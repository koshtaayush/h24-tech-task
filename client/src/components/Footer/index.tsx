//Library
import React from 'react'
import styled from 'styled-components'

interface Props {}

const Footer: React.FC<Props> = (props) => {
    return (
        <Container>
            Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und
            Versandkosten.
        </Container>
    )
}

export default Footer

const Container = styled.div`
    background-color: lightblue;
    text-align: center;
`
