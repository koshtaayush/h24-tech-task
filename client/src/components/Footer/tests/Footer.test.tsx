import React from 'react'
import { shallow } from 'enzyme'

import Footer from '..'

describe('Footer', () => {
    it('Should match the snapshot', () => {

        const component = shallow(
            <Footer />
        )

        expect(component).toMatchSnapshot()
    })
})
