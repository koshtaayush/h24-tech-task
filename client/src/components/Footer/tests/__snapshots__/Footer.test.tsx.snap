// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match the snapshot 1`] = `
<styled.div>
  Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und Versandkosten.
</styled.div>
`;
